### ISSUE TITLE
Módulo de UserService confuso

### ISSUE THREAD

**Issue by lpcinelli** [alissonfreire/octos-challenge#8]
O módulo de UserService dá para entender, mas está um pouco confuso

**Comment by alissonfreire** [comment]
Certo, movi as funções do service para módulos especializados e expus as funções em um top module Users
